content/intro/overview/what-is-cloudgov.md

```diff
@@ -22,3 +22,32 @@ A byproduct of cloud.gov’s creation is [compliance-masonry](https://github.com
 - Whether cloud.gov can broker self-service access to other third-party tools.
 - The need for federally-developed domain microservices around identity, authorization, etc.
 - What is needed for "Government-as-a-Platform" services, including PaaS-related container and clustering technologies, and whether to incorporate these into our toolbox.
+
+### What is your responsibility
+cloud.gov is a FedRAMP compliant Platform as a Service (Paas).  What this means is that agencies can deploy their application on a FedRAMP compliant platform and just be responsible app code.  The way it works is cloud.gov’s compliance is encapsulated within a boundary.  All components within that boundary are cloud.gov’s responsibility in terms of maintaining and operating the platform.  The specific items that cloud.gov is responsible for are the following:
+- Infrastructure
+- Supported Buildpacks
+- OS
+- Anti-Malware
+- Network Security 
+- Continuous Monitoring 
```

Comment:
For all of these we should say "Platform..." so "Platform Cotinuous monitoring", "Platform versioning" etc